packages/app/src/systems/Core/hooks/useTokensMethods.ts

```diff
@@ -20,7 +20,7 @@ export function useTokenMethods(tokenId: string) {
     },
     queryNetworkFee() {
       return contract.prepareCall.mint({
-        variableOutputs: 1,
+        variableOutputs: 2,
```

Comment:
Here also should be 1, as mint function only outputs 1 type of asset

Reply:
✅